RecoEgamma/EgammaElectronProducers/python/lowPtGsfElectronSequence_cff.py

```diff
@@ -0,0 +1,88 @@
+import FWCore.ParameterSet.Config as cms
+
+#==============================================================================
+# Sequence to make low pT electrons.
+# Several steps are cloned and modified to 'open up' the sequence:
+# tracker-driven electron seeds, KF track candidates, GSF tracks.
+#==============================================================================
+
+# Low pT electron seeds
+# Below relies on default configuration for generalTracks
+from RecoEgamma.EgammaElectronProducers.lowPtGsfElectronSeeds_cfi import *
+
+# Electron (KF) track candidates
+from TrackingTools.GsfTracking.CkfElectronCandidateMaker_cff import *
+TrajectoryFilterForElectronsOpen = TrajectoryFilterForElectrons.clone()
+TrajectoryFilterForElectronsOpen.minPt = 0.
+TrajectoryFilterForElectronsOpen.minimumNumberOfHits = 3
+TrajectoryBuilderForElectronsOpen = TrajectoryBuilderForElectrons.clone()
+TrajectoryBuilderForElectronsOpen.trajectoryFilter.refToPSet_ = 'TrajectoryFilterForElectronsOpen'
+electronCkfTrackCandidatesOpen = electronCkfTrackCandidates.clone()
+electronCkfTrackCandidatesOpen.TrajectoryBuilderPSet.refToPSet_ = 'TrajectoryBuilderForElectronsOpen'
+electronCkfTrackCandidatesOpen.src = 'lowPtGsfElectronSeeds'
+
+# GSF tracks
+from TrackingTools.GsfTracking.GsfElectronGsfFit_cff import *
+GsfElectronFittingSmootherOpen = GsfElectronFittingSmoother.clone()
+GsfElectronFittingSmootherOpen.ComponentName = 'GsfElectronFittingSmootherOpen'
+GsfElectronFittingSmootherOpen.MinNumberOfHits = 2
```

Comment:
@Sam-Harper @mverzett This nonstandard parameter value increases the eff of the TrkCandidate->GsfTrack step (based on generalTracks regardless of match to gen ele) from 84->91%. Not sure if we should do this or not ...

Reply:
I support this, we just have to be careful it doesnt blow CPU